core/server/common/src/main/java/alluxio/master/AbstractPrimarySelector.java

```diff
@@ -50,6 +51,8 @@
   private State mState = State.SECONDARY;
 
   protected final void setState(State state) {
+    AlluxioEvent.MasterIsTransitioning
+        .fire(String.format("Old-State: %s, New-State: %s", mState, state));
```

Comment:
fire already joins multiple arguments, so should this be split up into multiple strings?